evalml/tests/automl_tests/test_automl.py

```diff
@@ -1858,7 +1863,11 @@ def test_automl_error_callback_silent(mock_fit, mock_score, X_y_binary, caplog):
     caplog.clear()
     automl = AutoMLSearch(X_train=X, y_train=y, problem_type="binary", error_callback=silent_error_callback, train_best_pipeline=False, n_jobs=1)
     automl.search()
-    assert msg not in caplog.text
+    traceback_string1 = "return self._execute_mock_call(*args, **kwargs)"
+    traceback_string2 = "return self._mock_call(*args, **kwargs)"
+    traceback_string3 = "cv_pipeline.fit(X_train, y_train)"
+    for messages in [traceback_string1, traceback_string2, traceback_string3, msg]:
+        assert messages not in caplog.text
```

Comment:
Only `silent_error_callback` doesn't log the traceback

Reply:
It's confusing though cause there is no traceback since nothing errors. I think we should update the test. Could be done in a separate PR but it'd be a small change.

Reply:
@bchen1116 @freddyaboulton if an issue is coming out of this, can we also maybe parameterize these tests into one?  Ping me if you do it in a separate PR, I'd like to make sure it doesn't get lost in the backlog.

Reply:
@freddyaboulton @chukarsten simplified the tests, so I don't think we'll need to file that in a new issue.